spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -46,64 +46,70 @@ def assert_ems
+      :dns_policy     => "ClusterFirst",
+    )
 
     # Check the relation to container node
     @containergroup.container_node.should have_attributes(
-                                              :ems_ref => "58ffddce-e385-11e4-9d96-f8b156af4ae1"
-                                          )
+      :ems_ref => "248c52a3-286d-11e5-b5bb-727174f8ab71"
+    )
   end
 
   def assert_specific_container_node
     @containernode = ContainerNode.first
     @containernode.should have_attributes(
-                              :ems_ref       => "58ffddce-e385-11e4-9d96-f8b156af4ae1",
-                              :name          => "dhcp-0-167.tlv.redhat.com",
-                              :lives_on_type => nil,
-                              :lives_on_id   => nil
-                          )
+      :ems_ref       => "248c52a3-286d-11e5-b5bb-727174f8ab71",
+      :name          => "dhcp-0-129.tlv.redhat.com",
+      :lives_on_type => nil,
+      :lives_on_id   => nil
+    )
   end
 
   def assert_specific_container_service
     @containersrv = ContainerService.find_by_name("frontend")
     @containersrv.should have_attributes(
-                             :ems_ref          => "ff7e7aa3-e385-11e4-9d96-f8b156af4ae1",
-                             :name             => "frontend",
-                             :session_affinity => "None",
-                             :portal_ip        => "172.30.208.102"
-                         )
+      :ems_ref          => "f3a3e170-2870-11e5-b5bb-727174f8ab71",
+      :name             => "frontend",
+      :session_affinity => "None",
+      :portal_ip        => "172.30.187.127"
+    )
   end
 
   def assert_specific_container_project
-    @container_pr = ContainerProject.find_by_name("openshift")
+    @container_pr = ContainerProject.find_by_name("test")
     @container_pr.should have_attributes(
-                             :ems_ref => "581874d7-e385-11e4-9d96-f8b156af4ae1",
-                             :name    => "openshift"
-                         )
+      :ems_ref      => "9f2f3e05-286d-11e5-b5bb-727174f8ab71",
+      :name         => "test",
+      :display_name => ""
+    )
   end
 
   def assert_specific_container_route
     @container_route = ContainerRoute.find_by_name("route-edge")
     @container_route.should have_attributes(
-                                :ems_ref      => "ff8a8e45-e385-11e4-9d96-f8b156af4ae1",
-                                :name         => "route-edge",
-                                :host_name    => "www.example.com",
-                                :service_name => "frontend"
-                            )
+      :ems_ref      => "f3b59d42-2870-11e5-b5bb-727174f8ab71",
+      :name         => "route-edge",
+      :host_name    => "www.example.com",
+      :service_name => "frontend",
+    )
+
+    @container_route.container_project.should have_attributes(
+      :ems_ref => "9f2f3e05-286d-11e5-b5bb-727174f8ab71",
+      :name    => "test"
+    )
```

Comment:
Indeed. I added this test to check the container project <-> container route relation (was added in another PR, but as I changed the route spec test already, I added this to make sure everything is in order).